docs/sql/postgresql/enrollment/migration_1.7.0_1.8.0.sql

```diff
@@ -0,0 +1,3 @@
+-- Changeset enrollment-server/1.8.x/20240620-add-resultTexts.xml::1::Lubos Racansky
+-- Add result_texts column
+ALTER TABLE es_operation_template ADD result_texts TEXT;
```

Comment:
Do we really need a `TEXT` data type? The texts are messages displayed on a mobile phone, so a long varchar should be sufficient.

Reply:
🤔 A good point. Discussed with @petrdvorak. Since it is structured JSON, we prefer `TEXT`.